### ISSUE TITLE
receiving no reply

### ISSUE THREAD

**Issue by vvattap** [cis3296s22/patreon-discord-announcer#88]
After adding and authorizing the bot to run on the server, the command I tried to run wasn't creating any replies. When I checked the member list, it said the bot was offline. The command I ran was /help.

**Comment by alyona-egorova** [comment]
Yes! This bot seems to always be offline. Is it still working?

**Comment by BrailleBennett** [comment]
Yeah I really hope the dev brings the bot back online, it is literally exactly what I'm looking for.
